packages/components/chip/src/Chip.stories.tsx

```diff
@@ -152,13 +172,37 @@ export const UnionFilter: StoryFn = _args => {
   )
 }
 
-export const IntersectionFilter: StoryFn = _args => {
-  const [activeFilters, setActiveFilters] = useState<string[]>(['number', 'symbol'])
+const multipleintersectionSet = Array.from(
+  new Map(
+    Object.entries({
+      '🦍': ['land', 'wild'],
+      '🦒': ['land', 'wild'],
+      '🦌': ['land', 'wild'],
+      '🐄': ['land', 'domestic'],
+      '🐕': ['land', 'domestic'],
+      '🐬': ['sea', 'wild'],
+      '🐓': ['land', 'domestic'],
+      '🐿️': ['land', 'wild'],
+      '🦧': ['land', 'wild'],
+      '🐳': ['sea', 'wild'],
+      '🦈': ['sea', 'wild'],
+      '🦭': ['sea', 'land', 'wild'],
+      '🦆': ['sea', 'land', 'air', 'wild'],
+      '🐡': ['sea', 'wild'],
+      '🐟': ['sea', 'wild'],
+      '🐞': ['land', 'air', 'wild'],
+      '🐢': ['land', 'sea', 'wild', 'domestic'],
+      '🦜': ['land', 'air', 'wild', 'domestic'],
+    })
+  )
+).sort(() => 0.5 - Math.random())
```

Comment:
We should keep in mind people will read the code ("show source") so we should keep it simple. Can you remove the random ? I think it will also be an issue for chromatic.

Reply:
I think we don't need to use a combination of `Array.from`, `Math.random`, `new Map` and `Object.entries`, we could just have this: 
```
const animals = [
  { emoji: '🦍', tags: ['land', 'wild']},
  { emoji: '🦒', tags: ['land', 'wild']},
  { emoji: '🦌', tags: ['land', 'wild']},
  { emoji: '🐄', tags: ['land', 'domestic']},
  { emoji: '🐕', tags: ['land', 'domestic']},
  { emoji: '🐬', tags: ['sea', 'wild']},
  { emoji: '🐓', tags: ['land', 'domestic']},
  { emoji: '🐿️', tags: ['land', 'wild']},
  { emoji: '🦧', tags: ['land', 'wild']},
  { emoji: '🐳', tags: ['sea', 'wild']},
  { emoji: '🦈', tags: ['sea', 'wild']},
  { emoji: '🦭', tags: ['sea', 'land', 'wild']},
  { emoji: '🦆', tags: ['sea', 'land', 'air', 'wild']},
  { emoji: '🐡', tags: ['sea', 'wild']},
  { emoji: '🐟', tags: ['sea', 'wild']},
  { emoji: '🐞', tags: ['land', 'air', 'wild']},
  { emoji: '🐢', tags: ['land', 'sea', 'wild', 'domestic']},
  { emoji: '🦜', tags: ['land', 'air', 'wild', 'domestic']},
]

```

Maybe we can also only keep 2 examples (single selection and multiple selection)